pennylane_qsharp/device.py

```diff
@@ -125,7 +128,7 @@ def apply(self, operation, wires, par):
     def post_apply(self): #pragma no cover
         """Compile the Q# program"""
         for e in self.obs_queue:
-            self.measure += "set resultArray[{wires[0]}] = ".format(wires=e.wires)
+            self.measure += "set resultArray w/= {wires[0]} <- ".format(wires=e.wires)
```

Comment:
Did they also change the array syntax?

Reply:
Correct, I received warnings related to arrays being immutable and that this would be the way to go